thrift0.16/build.gradle

```diff
@@ -1,25 +1,19 @@
 // This module builds and publishes 'armeria-thrift0.16', which is compiled with
 // the source code of ':thrift0.13', ':thrift0.14.0', and ':thrift0.15.0'.
-final def THRIFT_VERSION = '0.16.0'
 
 dependencies {
-    api("org.apache.thrift:libthrift") {
-        version {
-            // Will fail the build if the override doesn't work
-            strictly THRIFT_VERSION
```

Comment:
Previously, it was possible because one of the multiple versions was set as `constraints`.
But now `thrift0.13` must not use `libthrift:0.16`. Anyway, it makes sense to set `strictly` to the thrift versions in `dependencies.toml`.